platforms/fabric/src/main/java/com/dfsek/terra/fabric/mixin/lifecycle/client/MinecraftClientMixin.java

```diff
@@ -34,6 +43,23 @@
 
 @Mixin(MinecraftClient.class)
 public class MinecraftClientMixin {
+    @Shadow
+    @Final
+    private static Logger LOGGER;
+    
+    @Redirect(method = "method_40187(Lnet/minecraft/util/registry/DynamicRegistryManager;Lnet/minecraft/world/gen/GeneratorOptions;" +
+                       "Lnet/minecraft/world/level/LevelInfo;Lnet/minecraft/resource/ResourceManager;" +
+                       "Lnet/minecraft/resource/DataPackSettings;)Lcom/mojang/datafixers/util/Pair;",
+              at = @At(value = "INVOKE",
+                       target = "net/minecraft/util/registry/DynamicRegistryManager.createAndLoad ()" +
+                                "Lnet/minecraft/util/registry/DynamicRegistryManager$Mutable;"))
+    private static Mutable injectBiomes() {
+        Mutable mutable = DynamicRegistryManager.createAndLoad();
+        LOGGER.info("Injecting Terra biomes...");
+        FabricUtil.registerBiomes(mutable.get(Registry.BIOME_KEY));
```

Comment:
Should this not dispatch the `BiomeRegistrationEvent` event?

Reply:
No. That event is no longer used, I've yet to remove it.

Reply:
I thought you said no API changes?

Is that not an api change?

Reply:
The Fabric platform implementation is not part of the API, if people are depending on platform components (which they shouldn't be) they should expect breaking changes between game versions.

Reply:
Alright